java/org/apache/coyote/Request.java

```diff
@@ -95,6 +95,7 @@ public Request() {
 
     // remote address/host
     private final MessageBytes remoteAddrMB = MessageBytes.newInstance();
+    private final MessageBytes peerAddrMB = MessageBytes.newInstance();
```

Comment:
Do we care that it is either `remote` or `peer`? So one of them wastes memory. Or it is negligible ?

Reply:
I think in many cases where one would need peer, remote can ba ueseful as well.